scripts/sync-algolia-index.ts

```diff
@@ -111,8 +115,8 @@ const generateAlgoliaRecords: () => AlgoliaRecord[] = () => {
 
 const syncAlgoliaIndex = async () => {
   const client = algoliasearch(
-    process.env.ALGOLIA_PROJECT,
-    process.env.AGOLIA_WRITE_KEY,
+    process.env.ALGOLIA_PROJECT ?? "",
+    process.env.AGOLIA_WRITE_KEY ?? "",
```

Comment:
Typescript finally checks this file. It complained about assigning `undefined` to `string`. Might be worth adding `envalid` and checking vars there for a better error message. I can do that in a follow-up.